db/schema.rb

```diff
@@ -175,6 +174,12 @@
     t.index ["brand_id"], name: "index_twitter_trackers_on_brand_id", unique: true, using: :btree
   end
 
+  create_table "user", force: :cascade do |t|
```

Comment:
`:cascade` isn't the issue. It's that it's creating a `user` table. We already have a `users` table. This has something to do with your local database being messed up. This should be removed.